MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -394,6 +408,10 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
 					_texCoordTL, 
 					_texCoordBR);			
 			
+		    unchecked
+		    {
+                ++_drawCount;
```

Comment:
the ++x notation is more native for processors than x++. I readed it in the Marc Gregoire book Professional C++. If @tomspilman or @KonajuGames says I change it like you said.